engine/api/auth/ldapclient.go

```diff
@@ -302,8 +310,8 @@ func (c *LDAPClient) Search(filter string, attributes ...string) ([]Entry, error
 
 func (c *LDAPClient) searchAndInsertOrUpdateUser(db gorp.SqlExecutor, username string) (*sdk.User, error) {
 	// Search user
-	search := fmt.Sprintf("(uid=%s)", username)
-	entry, errSearch := c.Search(search, "uid", "cn", "ou", "givenName", "sn", "mail")
+	search := fmt.Sprintf("(%s=%s)", c.conf.SearchAttributes.UserId, username)
+	entry, errSearch := c.Search(search, c.conf.SearchAttributes.UserId, c.conf.SearchAttributes.CommonName, c.conf.SearchAttributes.OrgUnit, c.conf.SearchAttributes.FirstName, c.conf.SearchAttributes.LastName, c.conf.SearchAttributes.Email)
```

Comment:
Shouldn't I check whether these variables are actually set and if not, set them to default values here?